.github/actions/charm-run-task/action.yaml

```diff
@@ -0,0 +1,27 @@
+name: 'Task Runner'
+description: 'Run tests for a charm'
+inputs:
+  charm-path:
+    description: 'Path to the charm'
+    required: true
+  test-type:
+    description: 'Test type (e.g., lint, fmt, etc.)'
+    required: true
+  tox-command:
```

Comment:
- `tox-args`?
- Why is this a required arg?
- We already have `test-type` so expecting the caller to pass `-vve lint` is redundant. The action should probably make use of `test-type` like so: `tox -e ${{ test-type }} {{ tox-args }}`.
- Why the `-vv` vs `-v` vs nothing is important? Why not set `-vv` everywhere? Future proofing for potential other tox args is likely [YAGNI](https://en.wikipedia.org/wiki/You_aren't_gonna_need_it).